packages/dashboard/src/features/analytics/components/AnalyticsConfigDialog.tsx

```
@@ -0,0 +1,127 @@
+            <MenuDialogHeader>
+              <MenuDialogTitle>Connection</MenuDialogTitle>
+              <MenuDialogCloseButton />
+            </MenuDialogHeader>
+
+            <MenuDialogBody>
+              <div className="flex flex-col gap-4">
+                <ReadOnlyField label="Host" value={connection.host} />
+                <ReadOnlyField label="Project ID" value={connection.posthogProjectId} />
+
+                <div className="flex flex-col gap-1.5">
+                  <label className="text-xs font-medium text-muted-foreground">Project API Key</label>
```

ESLint

[error] Replace `Project·API·Key` with `⏎····················Project·API·Key⏎··················`

(prettier/prettier)